frontend/src/components/VSearchResultsGrid/VImageGrid.vue

```diff
@@ -64,7 +61,9 @@ export default defineComponent({
       required: true,
     },
     fetchState: {
-      type: Object as PropType<FetchState<NuxtError>>,
+      type: Object as PropType<
```

Comment:
`VImageGrid` is used for search results that are updated in this PR, but also for the related images, which will be updated later, so we use both `FetchState` types here.